extensionsdk/Microsoft.Windows.DevHome.SDK/Microsoft.Windows.DevHome.SDK.idl

```diff
@@ -1643,5 +1669,16 @@ namespace Microsoft.Windows.DevHome.SDK
         IQuickStartProjectGenerationOperation CreateProjectGenerationOperation(String prompt, Windows.Storage.StorageFolder outputFolder);
     }
 
+    // Extensions can implement this provider to provide a way to
+    // create a project based on a prompt as part of the Dev Home 
+    // quick start project feature.
+    [experimental]
+    [contract(Microsoft.Windows.DevHome.SDK.DevHomeContract, 8)]
+    interface IQuickStartProjectProvider2 requires IQuickStartProjectProvider
+    {
+        // Used to create the chat style messages between the user and the chat UI.
```

Comment:
This comment can replace the comment above.

Reply:
I moved the comment to replace the above comment for IQuickstartProjectProvider2.